packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -1303,27 +1311,44 @@ class SlideshowView extends GalleryComponent {
   //-----------------------------------------| REACT |--------------------------------------------//
 
   blockAutoSlideshowIfNeeded(props = this.props) {
-    const { isGalleryInHover } = props;
-    const { pauseAutoSlideshowClicked, shouldBlockAutoSlideshow, isInView } =
+    const { isGalleryInHover, styleParams } = props;
+    const { pauseAutoSlideshowClicked, shouldBlockAutoSlideshow, isInView,isInFocus } =
       this.state;
     let should = false;
     if (!isInView || pauseAutoSlideshowClicked) {
       should = true;
     } else if (
-      isGalleryInHover &&
-      props.styleParams.pauseAutoSlideshowOnHover
+      styleParams.pauseAutoSlideshowOnHover
     ) {
-      should = true;
+      should = isGalleryInHover || isInFocus && styleParams.isAccessible;
     }
     if (shouldBlockAutoSlideshow !== should) {
       this.setState({ shouldBlockAutoSlideshow: should }, () => {
-        this.startAutoSlideshowIfNeeded(props.styleParams);
+        this.startAutoSlideshowIfNeeded(styleParams);
       });
     } else {
       return;
     }
   }
 
+  onFocus(){
```

Comment:
onFocus will trigger this element or any child of this element is in focus and that is why we don't need to specifically check if the items, arrows, or buttons are in focus, we just need to be careful with that

Reply:
👌